graphics/src/Material_TEST.cc

```diff
@@ -38,6 +39,12 @@ TEST_F(MaterialTest, Material)
   EXPECT_STREQ("/path/../materials/textures/texture_image",
                mat.TextureImage().c_str());
 
+  std::string filenameIn = std::string(PROJECT_SOURCE_PATH) +
```

Comment:
We should probably use `ignition::common::jointPaths` here.

Reply:
@JShep1 ping here too

Reply:
You might want to double check my update to the tests, specifically changing `/path` to` `path`, does common have a way of specifying root, prepending `/` before any sort of path?